doc/getting-started/sample-app.rst

```diff
@@ -440,56 +440,19 @@ Copy Files from the Development Computer to Your Target System
 1. Copy all the files generated on the development computer to the
    target system. This includes the sample application executable files,
    HMI_VM and RT_VM images, Debian packages for the Service VM and
-   Hypervisor, launch scripts, and the iasl tool built following the
-   Getting Started Guide. You can use ``scp`` to copy across the local network,
-   or use a USB stick:
+   Hypervisor, and the launch scripts. 
 
-   Option 1: use ``scp`` to copy files over the local network
-     Use ``scp`` to copy files from your development computer to the
-     ``~/acrn-work`` directory on the target (replace the IP address used in
-     this example with the target system's IP address you found earlier)::
+   Use ``scp`` to copy files from your development computer to the
+   ``~/acrn-work`` directory on the target (replace the IP address used in
+   this example with the target system's IP address you found earlier)::
 
-       cd ~/acrn-work
-
-       scp acrn-hypervisor/misc/sample_application/image_builder/build/*_vm.img \
-           acrn-hypervisor/build/acrn-my_board-MyConfiguration*.deb \
-           *acrn-service-vm*.deb MyConfiguration/launch_user_vm_id*.sh \
-           acpica-unix-20210105/generate/unix/bin/iasl \
-           acrn@10.0.0.200:~/acrn-work
-
-     Then on the target system, run these commands::
-
-       sudo cp ~/acrn-work/iasl /usr/sbin
-       sudo ln -s /usr/sbin/iasl /usr/bin/iasl
-
-   Option 2: use a USB stick to copy files
-     Because the VM image files are large, format your USB stick with a file
-     system that supports files greater than 4GB: extFAT or NTFS, but not FAT32.
-
-     Insert a USB stick into the development computer and run these commands::
-
-        disk="/media/$USER/"$(ls /media/$USER)
-
-        cd ~/acrn-work
-        cp acrn-hypervisor/misc/sample_application/image_builder/build/*_vm.img rt_vm.img "$disk"
-        cp acrn-hypervisor/build/acrn-my_board-MyConfiguration*.deb "$disk"
-        cp *acrn-service-vm*.deb "$disk"
-        cp MyConfiguration/launch_user_vm_id*.sh "$disk"
-        cp acpica-unix-20210105/generate/unix/bin/iasl "$disk"
-        sync && sudo umount "$disk"
-
-     Move the USB stick you just used to the target system and run
-     these commands to copy the files locally::
+     cd ~/acrn-work
 
-        disk="/media/$USER/"$(ls /media/$USER)
+     scp acrn-hypervisor/misc/sample_application/image_builder/build/*_vm.img \
+         acrn-hypervisor*.deb *acrn-service-vm*.deb \
```

Comment:
Don't you need to copy and install packages, such as acrn-devicemodel, acrnd, grub-acrn, and more, created by the debian_build.sh script?

Reply:
```
     cd ~/acrn-work

     scp acrn-hypervisor/misc/sample_application/image_builder/build/*_vm.img \
         acrn-hypervisor*.deb *acrn-service-vm*.deb \
         MyConfiguration/launch_user_vm_id*.sh \
         acrn@10.0.0.200:~/acrn-work

```
Above are command lines for copied files to install ACRN and launch VM.

Reply:
OK.  So I think you're saying we only need to copy the one acrn-hypervisor Debian file because the others aren't changed from what was installed in the GSG instructions, right? We might mention that here.

Reply:
Yes, You're right. I'll add a mention there.